recipe/meta.yaml

```diff
@@ -72,11 +80,14 @@ requirements:
     - jsoncpp
     - xz
     - cgns
-    - qt
     - boost-cpp
     - pugixml
     - libtheora
     - gl2ps
+    - xorg-libxfixes                     # [linux]
```

Comment:
Interesting do we know why the CDT didn't work?

Reply:
@jakirkham, TBH, I wasn't sure that this was needed. At this point, I am taking shots in the dark. Because there are still issues with OpenGL, I followed [the instructions in the conda-forge docs](https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl).

See my comment here https://github.com/conda-forge/paraview-feedstock/pull/126#issuecomment-850240209.